runbotics/runbotics-scheduler/src/migrations/1732517789213-ProcessesTabAccess.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ProcessesTabAccess1732517789213 implements MigrationInterface {
+    name = 'ProcessesTabAccess1732517789213';
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.manager.getRepository('feature_key').insert({ name: 'ALL_PROCESSES_READ' });
+        await queryRunner.manager.getRepository('authority_feature_key').insert({ authority: 'ROLE_ADMIN', feature_key: 'ALL_PROCESSES_READ' });
```

Comment:
to discuss if for ROLE_ADMIN this feature key is needed now

Reply:
it's not, removed